README.md

```diff
@@ -12,7 +12,7 @@ In the section `environment`, set `CHUNK_NUMBER` to the chunk that you are editi
 
     environment:
       - SOUNDS_PER_PAGE=10
-      - ANNOTATION_TASKS=mood,miscellaneous
+      - ANNOTATION_TASKS=mood,miscellaneous,mood-miscellaneous,all
```

Comment:
because these instructions will be for the class we should only list the required tasks here. is that just `mood-miscellaneous`?

Reply:
@dbogdanov are these the instructions the instership annotators follow or are they only for the class?
 Because in the first case we may also want them to annotate one task only or all of them.
In that case I think it is simpler to show all the options in the home page of the annotator and instruct them which one to go for dependeing on each case.

Reply:
By default, we want people to work on `mood-miscellaneous`. This covers both our annotators and the AMPLab assignment.

Reply:
It makes sense to add more information to the README. Perhaps, a bullet list with all annotation layouts, explaining the meaning of each?